zeitschrift-fur-kunstgeschichte.csl

```diff
@@ -47,8 +51,8 @@
     </names>
   </macro>
   <macro name="editor">
-    <names variable="editor translator" delimiter=", ">
-      <name delimiter="" prefix=" " and="text" delimiter-precedes-et-al="never" delimiter-precedes-last="never"/>
+    <names variable="editor translator" delimiter=". ">
```

Comment:
The `delimiter=". "` looks wrong for me, but I don't have a counter example. It would result in something like "Editor1. Editor2. Editor3 and Editor4". However, commas would look better.